sceptre/scipool/templates/sc-product-s3-private-enc.j2

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: S3 private encrypted ServiceCatalog product. (fdp-1p4da46rq)
+Parameters:
+  ProductName:
+    Type: String
+    Description: Name of the product that will be visible to the end user
+    Default: 'S3: Private Encrypted Bucket'
+Resources:
+  scs3product:
+    Type: AWS::ServiceCatalog::CloudFormationProduct
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E3002  # missing `owner` property error -> it's inside of included products.yaml
+            - E3003  # invalid property `Fn::Transform` error -> cfn-lint bug
+    Properties:
+      Name: !Ref ProductName
+      Description: This product builds an AWS S3 bucket encrypted with private
+        access accessible from any source.
+      ProvisioningArtifactParameters:
+        {{ sceptre_user_data.ProvisioningArtifactParameters|indent(8) }}
+      'Fn::Transform':
+        Name: 'AWS::Include'
+        Parameters:
+          Location: "s3://cfn-snippets-bucket-cloudformationsnippetsbucket-elu83sv8ocdz/scipool/products.yaml"
+      ReplaceProvisioningArtifacts: true
+  Associates3:
+    Type: AWS::ServiceCatalog::PortfolioProductAssociation
+    Properties:
+      PortfolioId: !ImportValue
+        'Fn::Sub': '${AWS::Region}-sc-portfolio-s3-basic-SCS3portfolioId'
+      ProductId: !Ref scs3product
+  constraintec2linux:
```

Comment:
It looks like the launch role and portfolio id are correct... this constraint should be named differently though, for clarity.

Reply:
this is the same as before this refactor so how about leave this for a follow on PR?